apps/src/applab/sanitizeHtml.js

```diff
@@ -47,10 +47,29 @@ function warnAboutUnsafeHtml(warn, unsafe, safe, warnings) {
     return 0;
   };
 
+  // Do not warn when these attributes are removed.
+  var ignoredAttributes = ['kl_vkbd_parsed'];
```

Comment:
Should `pmbx_context` also be in this list?

Might also be worth a short comment in this file explaining that this is some mysterious attribute that we dont know much about, but know that we're fine with removing it silently.